### PR TITLE
Dockerize Telemetry Proxy

### LINKED ISSUE #42: Store gRPC telemetry data in database
Add GoLang backend docker container to receive telemetry streaming and populate influxdb

Need to add second grpc server endpoint and start server->client stream for telemetry

### PR DESCRIPTION
**Pull request by ZachGarcia42**
Fixes #42

### CODE REVIEW

**Comment by ZachGarcia42** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/73?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#73** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/73?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#72** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/72?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#71** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/71?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ZachGarcia42 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: telemetry-proxy/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Need ubuntu for dynamic linking
+FROM ubuntu:22.04 AS run
+
+# Copy the build output from the GitHub Workflow to Alpine
```
Comment (maxslarsson) [inline telemetry-proxy/Dockerfile:4]: Idk how much it matters if we have the right comment, so feel free to just resolve this and I'll approve if you don't want to change it :)

**Review by maxslarsson — CHANGES_REQUESTED** [review]

File: telemetry-proxy/Dockerfile (hunk 1 above)
Comment (ZachGarcia42) [reply telemetry-proxy/Dockerfile]: Lol, it's worth doing it right I think. Updated

**Review by maxslarsson — APPROVED** [review]

**Comment by ZachGarcia42** [comment]
### Merge activity

* **Sep 30, 1:49 PM EDT**: @ZachGarcia42 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/73).

**Pull request merged**